var/spack/repos/builtin/packages/melissa/package.py

```diff
@@ -11,14 +11,12 @@ class Melissa(CMakePackage):
     """
 
     homepage = "https://gitlab.inria.fr/melissa/melissa"
-    url = "https://gitlab.inria.fr/melissa/melissa/-/archive/v1.0/melissa-v1.0.tar.bz2"
     git = "https://gitlab.inria.fr/melissa/melissa.git"
-
+    url = "https://gitlab.inria.fr/melissa/melissa.git"
```

Comment:
A `git` url like this is not going to work with version 0.7.1 which expects an archive. Please test that older versions are still downloading, even if deprecated.